torchbenchmark/models/pytorch_unet/__init__.py

```diff
@@ -6,7 +6,6 @@
 from torch import optim
 from typing import Tuple
 
-torch.backends.cudnn.deterministic = True
 torch.backends.cudnn.benchmark = False
```

Comment:
I think we should also remove this line, to keep a consistent behavior with upstream.